OvaryModelGenerator/PelvisModelGenerator.cxx

```diff
@@ -16,25 +16,16 @@
 
 PelvisModelGenerator::PelvisModelGenerator(){
 
-  this->intercristalDistance = 1;
-  this->pelvisScaleFactor = 1;
+  this->m_IntercristalDistance = 1;
```

Comment:
In this file most of changes are variable name changes. See header file comment for the reason.